src/slate-toolbar/slate-toolbar.stories.js

```diff
@@ -36,12 +36,26 @@ export const ThreeColumns = () => (
     />
 );
 
+const order = [
+        "fontDecrease", "bold", "italic", "underlined", "deleted", "code", "superscript", "subscript", "color",
+        "fontIncrease", "heading1", "heading2", "heading3", "block-quote", "bulleted-list", "ordered-list", "image", "link"
+      ];
+
 export const Ordered = () => (
   <SlateToolbar
     orientation="vertical"
     colors={{ background: "#177991", fill: "#ffffff" }}
-    buttonsPerRow={5}
-    order={["fontDecrease", "bold", "italic", "underlined", "deleted", "code", "superscript", "subscript", "color",
-            "fontIncrease"]}
```

Comment:
Removing this means we no longer have a storybook story where we handle an ordered list that doesn't include ALL of the toolbar buttons (and shows how to exclude buttons from the toolbar).  Might be worth sticking in one more storybook story in this module that passes in a list to the `SlateToolbar` component which contains a subset of the buttons.